apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.html.ts

```diff
@@ -76,323 +76,265 @@ export function InstructorQuestionSettings({
-                                                  class="btn btn-sm btn-outline-dark dropdown-toggle"
-                                                  data-toggle="dropdown"
-                                                  aria-haspopup="true"
-                                                  aria-expanded="false"
-                                                >
-                                                  Add...
-                                                </button>
-                                                <div
-                                                  class="dropdown-menu"
-                                                  aria-labelledby="addSharingSet"
-                                                >
-                                                  ${sharingSetsOther.map(function (sharing_set) {
-                                                    return html`
-                                                      <button
-                                                        class="dropdown-item"
-                                                        type="submit"
-                                                        name="unsafe_sharing_set_id"
-                                                        value="${sharing_set.id}"
-                                                      >
-                                                        ${sharing_set.name}
-                                                      </button>
-                                                    `;
-                                                  })}
-                                                </div>
-                                              </div>
-                                            </form>
-                                          `
-                                        : ''}
-                                      <button
-                                        class="btn btn-sm btn-outline-primary"
-                                        type="button"
-                                        data-toggle="modal"
-                                        data-target="#publiclyShareModal"
-                                      >
-                                        Share Publicly
-                                      </button>
-                                      ${PubliclyShareModal({
-                                        csrfToken: resLocals.__csrf_token,
-                                        qid: resLocals.question.qid,
-                                      })}
-                                    `
-                                  : ''}
-                              `}
-                        </td>
-                      </tr>
+                      <hr />
+                      <div class="form-group">
+                        <div class="align-middle">Sharing</div>
+                        <div data-testid="shared-with">
+                          ${questionSharing({
```

Comment:
I moved the question sharing piece to a new "component" below to help with indentation and readability.

Reply:
Remember to use `PascalCase` here.

Reply:
Updated to `QuestionSharing` in [f9945bf](https://github.com/PrairieLearn/PrairieLearn/pull/9687/commits/f9945bf0e4c4e0d20d0a34f07c433d9e135dc0c7).